backend/src/main/resources/db/migration/V2__create_audit_tables.sql

```diff
@@ -0,0 +1,148 @@
+                                         street text,
+                                         substitute text,
+                                         title text,
+                                         website text
+);
+
+CREATE TABLE healthcare_provider_category_aud (
+                                                  rev integer NOT NULL,
+                                                  institution_id bigint NOT NULL,
+                                                  location_id bigint NOT NULL,
+                                                  id bigint NOT NULL,
+                                                  revtype smallint
+);
+
+CREATE TABLE selfexamination_record_aud (
+                                            id bigint NOT NULL,
+                                            rev integer NOT NULL,
+                                            revtype smallint,
+                                            due_date date,
+                                            result text,
+                                            status text,
+                                            type text,
+                                            uuid text,
+                                            waiting_to date,
+                                            account_id bigint
+);
+
+ALTER TABLE ONLY account_aud
+    ADD CONSTRAINT account_aud_pkey PRIMARY KEY (id, rev);
+
+ALTER TABLE ONLY badge_aud
+    ADD CONSTRAINT badge_aud_pkey PRIMARY KEY (account_id, type, rev);
+
+ALTER TABLE ONLY examination_record_aud
+    ADD CONSTRAINT examination_record_aud_pkey PRIMARY KEY (id, rev);
+
+ALTER TABLE ONLY healthcare_category_aud
+    ADD CONSTRAINT healthcare_category_aud_pkey PRIMARY KEY (id, rev);
+
+ALTER TABLE ONLY healthcare_provider_aud
+    ADD CONSTRAINT healthcare_provider_aud_pkey PRIMARY KEY (institution_id, location_id, rev);
+
+ALTER TABLE ONLY healthcare_provider_category_aud
+    ADD CONSTRAINT healthcare_provider_category_aud_pkey PRIMARY KEY (rev, institution_id, location_id, id);
+
+ALTER TABLE ONLY selfexamination_record_aud
+    ADD CONSTRAINT selfexamination_record_aud_pkey PRIMARY KEY (id, rev);
+
+ALTER TABLE ONLY examination_record_aud
+    ADD CONSTRAINT fk5lxlg92fn2q6mul14ly8n5fbr FOREIGN KEY (rev) REFERENCES revinfo(rev);
+
+ALTER TABLE ONLY account_aud
+    ADD CONSTRAINT fkaexie5n0kol2mjlvo03ii45d0 FOREIGN KEY (rev) REFERENCES revinfo(rev);
+
+ALTER TABLE ONLY selfexamination_record_aud
+    ADD CONSTRAINT fkg1uyxqsvkobw6dnx74h1nj2x2 FOREIGN KEY (rev) REFERENCES revinfo(rev);
+
+ALTER TABLE ONLY healthcare_provider_category_aud
+    ADD CONSTRAINT fkivi6tglk4kbdxq4sa25n6s43f FOREIGN KEY (rev) REFERENCES revinfo(rev);
+
+ALTER TABLE ONLY healthcare_category_aud
+    ADD CONSTRAINT fkj0f3nuqp8iv9ms2w6h6025soe FOREIGN KEY (rev) REFERENCES revinfo(rev);
+
+ALTER TABLE ONLY badge_aud
+    ADD CONSTRAINT fkk8lubwsfxvhhsm8ttaou0m0yt FOREIGN KEY (rev) REFERENCES revinfo(rev);
```

Comment:
Maybe a better name? E.g. fk_badge_aud_revinfo. Same for the rest.

Reply:
Definitely, somehow I overlooked it, fixed in [25dd2b2](https://github.com/cesko-digital/loono-be/pull/73/commits/25dd2b2176cd6b8c32add45eabec2ced61971fd2)